docs/user/commercial/index.rst

```diff
@@ -15,24 +46,25 @@ Private repositories and private documentation
 Additional build resources
     Do you have a complicated build process that uses large amounts
     of CPU, memory, disk, or networking resources?
-    Our commercial offering has much higher default resources
-    that result in faster documentation build times
-    and we can increase it further for very demanding projects.
+    Commercial subscriptions offer more resources
+    which results in faster documentation build times.
+    Resources can be increased further for very demanding projects.
 
 Priority support
     We have a dedicated support team that responds to support requests during business hours.
-    If you need a quick turnaround, please signup for readthedocs.com.
 
 Advertising-free
     All commercially hosted documentation is always ad-free.
 
-.. _readthedocs.org: https://readthedocs.org
-.. _readthedocs.com: https://readthedocs.com
+Business features
+    A number of extra features exist specifically for business needs, such as
+    organization support with team management, single-sign on, and audit logging.
+
+    .. seealso::
 
-.. toctree::
-   :caption: Additional commercial features
+       - :doc:`/commercial/organizations`
+       - :doc:`/commercial/single-sign-on`
+       - :doc:`/security-log`
 
-   organizations
-   single-sign-on
-   sharing
-   privacy-level
+.. _readthedocs.org: https://readthedocs.org
+.. _readthedocs.com: https://readthedocs.com
```

Comment:
```suggestion
```

🔥

Reply:
Nice find! I kind of wish that Sphinx would have told me this...

Reply:
Yea, would really love a better rst/sphinx linter.

Reply:
I was wondering if it's possible to write an upstream issue quickly... but with 1.1k issues open (and 101 PRs) and a lack of response to critical bugs, I'm just not gonna do it :(